dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserController.java

```diff
@@ -558,6 +574,18 @@ protected void prePatchEntity( User entity ) throws Exception
         }
     }
 
+    @Override
+    protected void postPatchEntity( User entity )
```

Comment:
Try to separate fixes/features in individual PRs so that we can test and back-port to previous versions independently.